### ISSUE TITLE
In location based AR, I am not able to walk around the 3-D model.

### ISSUE THREAD

**Issue by vizsatiz** [AR-js-org/AR.js#566]
**Do you want to request a *feature* or report a *bug*?**
Report a bug

**What is the current behavior?**
When I create and run the AR.js app for goe-location-based AR model rendering, the model rendered stays there, and when I move towards it, it moves back. By this I mean, am not able to move around the model as the model keeps facing me. Is this expected ? I am trying out in Bangalore, India.

**If the current behavior is a bug, please provide the steps to reproduce.**

**Please mention other relevant information such as the browser version, Operating System and Device Name**
Tested this in chrome & safari on phone

**What is the expected behavior?**
I should be able to move around the model

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Comment by Platform-Group** [comment]
0.0001 degrees to the north is about 11 meters, so not much, if I had to guess it's simply because your GPS isn't updating your position because you're not walking very far? The cameras position can only be updated as the GPS is updated, if your GPS is relatively inaccurate then it's not going to pick up changes of only 11m. I know my current AR.js project is in a skyscraper so GPS is practically useless thanks to the interference and I had to put in a system for changing simulated coordinates.

**Comment by Platform-Group** [comment]
I forgot to mention, you can always put some logs into your gps-camera-update-position to see when it's firing and how accurate it is. Here's a function for measuring the distance in meters between two sets of coordinates, it may help you in debugging.

```
  function measure(lat1, lon1, lat2, lon2) {
    // generally used geo measurement function
    var R = 6378.137; // Radius of earth in KM
    var dLat = (lat2 * Math.PI) / 180 - (lat1 * Math.PI) / 180;
    var dLon = (lon2 * Math.PI) / 180 - (lon1 * Math.PI) / 180;
    var a =
      Math.sin(dLat / 2) * Math.sin(dLat / 2) +
      Math.cos((lat1 * Math.PI) / 180) *
        Math.cos((lat2 * Math.PI) / 180) *
        Math.sin(dLon / 2) *
        Math.sin(dLon / 2);
    var c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
    var d = R * c;
    return d * 1000; // meters
  }
```

**Comment by mamoone** [comment]
any good news ?!

**Comment by vizsatiz** [comment]
> 0.0001 degrees to the north is about 11 meters, so not much, if I had to guess it's simply because your GPS isn't updating your position because you're not walking very far? The cameras position can only be updated as the GPS is updated, if your GPS is relatively inaccurate then it's not going to pick up changes of only 11m. I know my current AR.js project is in a skyscraper so GPS is practically useless thanks to the interference and I had to put in a system for changing simulated coordinates.

I really dont think this is the issue as I tried changing those values

**Comment by vizsatiz** [comment]
> any good news ?!

All I could understand was there were some issues with Southern hemisphere, for more info https://github.com/AR-js-org/AR.js/issues/278

**Comment by nickw1** [comment]
I don't think this is anything to do with the Southern Hemisphere, but possibly inherent limitations in the sensors of specific devices and/or iOS-related bugs. Perhaps related to #547 #552 #590?

**Comment by marcusx2** [comment]
It's normal, you can't [walk around](https://github.com/AR-js-org/AR.js/issues/490) a model.

**Comment by Windows-server-2012631** [comment]
> > any good news ?!
> 
> All I could understand was there were some issues with Southern hemisphere, for more info [#278](https://github.com/AR-js-org/AR.js/issues/278)





https://lnkr.fun/c259a31

**Comment by nickw1** [comment]
Deleted reply with suspicious link and no content
